tests/integration/usage_statistics/conftest.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import requests
+from requests.adapters import HTTPAdapter, Retry
+
+
+@pytest.fixture(scope="session")
+def requests_session_with_retries() -> requests.Session:
+    # https://stackoverflow.com/a/35636367
+    session = requests.Session()
+    retries = Retry(total=5, backoff_factor=1, status_forcelist=[502, 503, 504])
```

Comment:
I like how the `backoff_factor` works! We should keep an eye on how often we are using this though for performance reasons.